apps/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -576,6 +577,34 @@ data(_) ->
     emqx_ctl:usage([{"data import <File>",   "Import data from the specified file"},
                     {"data export",          "Export data"}]).
 
+%%--------------------------------------------------------------------
+%% @doc acl Command
+
+acl(["cache-clean", "node", Node]) ->
+    case emqx_mgmt:clean_acl_cache_all(erlang:list_to_existing_atom(Node)) of
+        ok ->
+            emqx_ctl:print("ACL cache drain started on node ~s.~n", [Node]);
```

Comment:
May it be "Clean ACL cache for all of the clients"? As the "Drain started" would be another terminology the user need to learn.